revscoring/features/diff.py

```diff
@@ -8,9 +8,35 @@
 from .util import MARKUP_RE, NUMERIC_RE, SYMBOLIC_RE
 
 bytes_changed = revision.bytes - parent_revision.bytes
+"""
+Represents bytes changed in this edit.
```

Comment:
"encoded content bytes changed"  Might also be good to note that for text, this is usually UTF-8 encoded.